server/src/models/song.model.ts

```diff
@@ -0,0 +1,23 @@
+import { Schema, model, models } from 'mongoose';
+import { SongSchema } from '../types/song.types';
+
+const songSchema = new Schema<SongSchema>({
+  title: { type: String, required: true },
+  tempo: [{ type: String, required: true }],
+  originalKey: { type: String, required: true },
+  tags: [{ type: String, required: true }],
```

Comment:
themes, and type should be `Types.Array<String>` remember to import Types from mongoose

Reply:
hm? but but it's an array fo string right?